supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/Run.scala

```diff
@@ -92,11 +92,16 @@ case class Run(runId: String, genomicsInterface: Genomics) {
   def status(): RunStatus = {
     val op = genomicsInterface.operations().get(runId).execute
     if (op.getDone) {
-      // If there's an error, generate a Failed status. Otherwise, we were successful!
       val eventList = getEventList(op)
+      val ceInfo = op.getMetadata.get ("runtimeMetadata").asInstanceOf[GArrayMap[String,Object]].get("computeEngine").asInstanceOf[GArrayMap[String, String]]
```

Comment:
There's a lot of `getOrDefault`s below to cover us if they're missing. Which makes this combination of `get` and `asInstanceOf` look quite a bit more dangerous by comparison!